examples/events/basicevent/basicevent.go

```diff
@@ -0,0 +1,121 @@
+package basicevent
+
+import (
+	"errors"
+	"github.com/golang/glog"
+	"time"
+
+	"github.com/mishudark/triper"
+)
+
+
+var (
+	//ErrSubjectMissing When an event has no subject id
```

Comment:
an space is required between the `// ` and the comment

Reply:
👍 correcting it in all of the comments :)